### PR TITLE
Cards Demo Branch Merge 2

### PR DESCRIPTION
**Pull request by noodlemoodle**

### CODE REVIEW

File: src/foam/nanos/auth/LifecycleState.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ foam.ENUM({
     },
     {
       name: 'DISABLED',
-      label: 'Disabled'
+      label: 'Paused'
```
Comment (jlhughes) [inline src/foam/nanos/auth/LifecycleState.js]: Please revert this.  If you need a label change, it needs to be app specific.

**Review by sarthak-marwaha — APPROVED** [review]
LGTM

**Comment by thepinkcacao** [comment]
LGTM

File: src/foam/nanos/auth/LifecycleState.js (hunk 1 above)
Comment (noodlemoodle) [reply src/foam/nanos/auth/LifecycleState.js]: reverted and [ticketed](https://nanopay.atlassian.net/browse/NP-8461)

**Review by jlhughes — APPROVED** [review]

**Review by tharmaman — COMMENTED** [review]
Changes here should be fine.

**Pull request merged**